build.gradle

```diff
@@ -33,10 +33,9 @@ task preprocessContentAssets(type: Copy,
     group: "Build") {
     from contentDir
     into generatedAssetDir
-    filteringCharset 'UTF-8'
     includeEmptyDirs false
     
-    include "**/*.js", "**/.css",
+    include "**/*.js", "css/*.css",
```

Comment:
@vieiro I have this as `**/.css` to get all CSS, and I see a bug there you are pointing out where it should be `**/*.css` instead. There are different CSS files in the NB code base, and left it as this until we get everything moved over.

Reply:
Interestingly I thought Gj and I had fixed this while down in Atlanta, but perhaps we forgot to push changes from his laptop.